public_suffix_list.dat

```diff
@@ -12516,6 +12516,10 @@ ip.linodeusercontent.com
 // Submitted by Victor Velchev <admin@liquidnetlimited.com>
 we.bs
 
+// Localcert : https://localcert.dev
+// Submitted by Lann Martin <security@localcert.dev>
+user.localcert.dev
```

Comment:
Based on your description of the names, shouldn't this be

`*.user.localcert.dev`?

Reply:
Maybe? My intent is for `A.X.localcert.dev` and `A.Y.localcert.dev` to be in different "security domains". My understanding is that `*.user.localcert.dev` would put `A.X.localcert.dev` and `B.X.localcert.dev` in different domains as well, which I don't think is necessary in this case.

Reply:
After further review and testing I believe Let's Encrypt _will_ issue a cert for domains in the private section, so I don't think that is a concern here.

I would still slightly prefer the rule as-is; I believe it fits the purposes of the PSL and would be slightly more useful to users. But again, if there is still concern about it I'll make the change.